modules/ROOT/partials/glossary.adoc

```diff
@@ -1,117 +1,132 @@
-[[procParam]]$procParam:: Procedure parameter name.
-[[procParamFmt]]$procParamFmt:: Freeform procedure parameter format.
-[[propKey]]$propKey:: Property key name, for example, name.
-[[propKeyList]]$propKeyList:: Comma-separated list of property key names.
-[[query]]$query:: Query string extract, for example, `MATCH (n) WHERE n.prop...`.
-[[reasonList]]$reasonList:: Comma-separated list of reasons of the failure.
+//Change in the code to $input and update the notifications.
+[[proc]]$proc:: Procedure name, for example, `db.labels()`.
+[[procClass]]$procClass:: User-defined procedure implementation class name, for example, `MyProcedureClass`.
+[[procExeMode]]$procExeMode:: Procedure execution mode, one of `READ`, `WRITE`, `SCHEMA`, and `DBMS`.
+[[procField]]$procField:: User-defined procedure implementation class argument name, for example, `myArgument`.
+//Change $proField to $arg
+[[procFieldType]]$procFieldType:: User-defined procedure implmenetation class argument type, one of `input` or `output`.
+//Change $procFieldType to $argType
+[[procFun]]$procFun:: Procedure or function name or ID, for example, `db.labels()` or `collect()`.
+[[procMethod]]$procMethod:: User-defined procedure implmenetation class method name, for example, `myProcedureMethod`
+[[procParam]]$procParam:: User-defined procedure implmenetation parameter name.
+[[procParamFmt]]$procParamFmt:: Freeform user-defined procedure implmenetation parameter format, for example, `com.example.*`.
+[[propKey]]$propKey:: Property key name, for example, `name`.
+[[propKeyList]]$propKeyList:: A list of property key names, for example, `name, age, hometown`.
+[[query]]$query:: A statement that retrieves or writes information to a database, for example, `MATCH (n) WHERE n.prop`.
+[[reasonList]]$reasonList:: A list of reasons of the failure.
 [[relType]]$relType:: Relationship type name, for example, `KNOWS`.
-[[replacement]]$replacement:: Replacement
-[[role]]$role:: Role name, for example, `admin`.
+[[replacement]]$replacement:: A free-form text, a replacement of a deprecated feature, a Cypher command, etc.
+[[role]]$role:: RBAC role name, for example, `admin`, `public`, `receptionist`.
 [[routingPolicy]]$routingPolicy:: Routing policy name.
-[[runtime]]$runtime:: Cypher runtime name.
-[[schemaDescr]]$schemaDescr:: Schema descriptor.
-[[schemaDescrType]]$schemaDescrType:: Type of schema descriptor.
-[[selector]]$selector:: GPM path selector.
+[[runtime]]$runtime:: Cypher runtime name. One of `slotted`, `pipelined`, and `parallel`.
+[[schemaDescr]]$schemaDescr:: Schema descriptor, for example, `(:Label1 \{prop1})`.
+[[schemaDescrType]]$schemaDescrType:: Freeform type of schema descriptor, for example, `label property existence constraint`.
+[[selector]]$selector:: GPM path selector, for example, `ALL PATHS`, `SHORTEST 2`, etc.
 [[selectorType]]$selectorType:: Selector type.
+//Ask Lasse about $selectorType (releated to CDC) and has nothing to do with $selector.
```

Comment:
Do we still want these comments here or transferred to Trello/ a doc. I don't really mind as long as we clean them up when each comment is sorted

Reply:
Yeah, let's keep them for now. It'll be easier to find out later what needs to be updated.